hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -49,120 +49,128 @@
+        PS_INPUTMAPPING_EXPAND_NORMAL, // invalid for final combiner
+    PS_REGISTER_ONE_HALF =
+        PS_REGISTER_ZERO |
+        PS_INPUTMAPPING_HALFBIAS_NEGATE, // invalid for final combiner
+    PS_REGISTER_NEGATIVE_ONE_HALF =
+        PS_REGISTER_ZERO |
+        PS_INPUTMAPPING_HALFBIAS_NORMAL, // invalid for final combiner
 };
 
-enum PS_COMBINERCOUNTFLAGS
-{
-    PS_COMBINERCOUNT_MUX_LSB=     0x0000L, // mux on r0.a lsb
-    PS_COMBINERCOUNT_MUX_MSB=     0x0001L, // mux on r0.a msb
+enum PS_COMBINERCOUNTFLAGS {
+    PS_COMBINERCOUNT_MUX_LSB = 0x0000L, // mux on r0.a lsb
+    PS_COMBINERCOUNT_MUX_MSB = 0x0001L, // mux on r0.a msb
 
-    PS_COMBINERCOUNT_SAME_C0=     0x0000L, // c0 same in each stage
-    PS_COMBINERCOUNT_UNIQUE_C0=   0x0010L, // c0 unique in each stage
+    PS_COMBINERCOUNT_SAME_C0 = 0x0000L, // c0 same in each stage
+    PS_COMBINERCOUNT_UNIQUE_C0 = 0x0010L, // c0 unique in each stage
 
-    PS_COMBINERCOUNT_SAME_C1=     0x0000L, // c1 same in each stage
-    PS_COMBINERCOUNT_UNIQUE_C1=   0x0100L  // c1 unique in each stage
+    PS_COMBINERCOUNT_SAME_C1 = 0x0000L, // c1 same in each stage
+    PS_COMBINERCOUNT_UNIQUE_C1 = 0x0100L // c1 unique in each stage
 };
 
-enum PS_COMBINEROUTPUT
-{
-    PS_COMBINEROUTPUT_IDENTITY=            0x00L, // y = x
-    PS_COMBINEROUTPUT_BIAS=                0x08L, // y = x - 0.5
-    PS_COMBINEROUTPUT_SHIFTLEFT_1=         0x10L, // y = x*2
-    PS_COMBINEROUTPUT_SHIFTLEFT_1_BIAS=    0x18L, // y = (x - 0.5)*2
-    PS_COMBINEROUTPUT_SHIFTLEFT_2=         0x20L, // y = x*4
-    PS_COMBINEROUTPUT_SHIFTRIGHT_1=        0x30L, // y = x/2
+enum PS_COMBINEROUTPUT {
+    PS_COMBINEROUTPUT_IDENTITY = 0x00L, // y = x
+    PS_COMBINEROUTPUT_BIAS = 0x08L, // y = x - 0.5
+    PS_COMBINEROUTPUT_SHIFTLEFT_1 = 0x10L, // y = x*2
+    PS_COMBINEROUTPUT_SHIFTLEFT_1_BIAS = 0x18L, // y = (x - 0.5)*2
+    PS_COMBINEROUTPUT_SHIFTLEFT_2 = 0x20L, // y = x*4
+    PS_COMBINEROUTPUT_SHIFTRIGHT_1 = 0x30L, // y = x/2
 
-    PS_COMBINEROUTPUT_AB_BLUE_TO_ALPHA=    0x80L, // RGB only
+    PS_COMBINEROUTPUT_AB_BLUE_TO_ALPHA = 0x80L, // RGB only
 
-    PS_COMBINEROUTPUT_CD_BLUE_TO_ALPHA=    0x40L, // RGB only
+    PS_COMBINEROUTPUT_CD_BLUE_TO_ALPHA = 0x40L, // RGB only
 
-    PS_COMBINEROUTPUT_AB_MULTIPLY=         0x00L,
-    PS_COMBINEROUTPUT_AB_DOT_PRODUCT=      0x02L, // RGB only
+    PS_COMBINEROUTPUT_AB_MULTIPLY = 0x00L,
+    PS_COMBINEROUTPUT_AB_DOT_PRODUCT = 0x02L, // RGB only
 
-    PS_COMBINEROUTPUT_CD_MULTIPLY=         0x00L,
-    PS_COMBINEROUTPUT_CD_DOT_PRODUCT=      0x01L, // RGB only
+    PS_COMBINEROUTPUT_CD_MULTIPLY = 0x00L,
+    PS_COMBINEROUTPUT_CD_DOT_PRODUCT = 0x01L, // RGB only
 
-    PS_COMBINEROUTPUT_AB_CD_SUM=           0x00L, // 3rd output is AB+CD
-    PS_COMBINEROUTPUT_AB_CD_MUX=           0x04L, // 3rd output is MUX(AB,CD) based on R0.a
+    PS_COMBINEROUTPUT_AB_CD_SUM = 0x00L, // 3rd output is AB+CD
+    PS_COMBINEROUTPUT_AB_CD_MUX =
+        0x04L, // 3rd output is MUX(AB,CD) based on R0.a
```

Comment:
Also re-align.

With so many aligned stuff (these are literally tables translated into code-form), we should consider keeping the old enums in my opinion.